src/recovery/cubic.rs

```diff
@@ -235,7 +240,16 @@ fn on_packet_acked(
             cubic_cwnd = cmp::max(cubic_cwnd, lss_cwnd);
         }
 
-        r.congestion_window = cubic_cwnd;
+        // Update the increment and increase cwnd by MSS.
+        r.cubic_state.cwnd_inc += cubic_cwnd - r.congestion_window;
+
+        // cwnd_inc can be more than 1 MSS in the late stage of max probing.
+        // however QUIC recovery draft 7.4 (Congestion Avoidance) limit
```

Comment:
```suggestion
        // however QUIC recovery draft 7.4 (Congestion Avoidance) limits
```

Reply:
Also, this section seems to have been moved in the latest drafts, so this reference is not valid.